packages/model-viewer/src/three-components/Damper.ts

```diff
@@ -34,7 +31,11 @@ export class Damper {
 
   update(
       x: number, xGoal: number, timeStepMilliseconds: number,
-      xNormalization: number): number {
+      xNormalization: number, decayMilliseconds: number = 50): number {
+    const naturalFrequency =
+        1 / Math.max(MIN_DECAY_MILLISECONDS, decayMilliseconds);
```

Comment:
This is probably a good idea, but I'd still like to hear your reasoning on making 50 the minimum value. Also, this reminds me we should add documentation before we merge this, as details like this are important to communicate.

Reply:
You might consider parameterizing the damper with naturalFrequency directly, so that the max and divide can happen once instead of inside the loop.

Reply:
I made the default value 50 because this was the previous value, I added a default to not break any code that might be using the Damper. For the minimum I just picked an arbitrary small value to avoid a divide by 0 situation, since the change allows anyone to set this value, and this is one way it could go wrong.

Added setting the naturalFrequency to avoid doing the divide every frame.

Reply:
Oh, now I see. I missed that `MIN_DECAY_MILLISECONDS` was a different constant. Thanks.